src/lib/logger.coffee

```diff
@@ -52,17 +53,25 @@ exports.log = ->
 	publish(arguments...)
 
 do ->
+	_lock = new Lock()
```

Comment:
I've recently seen the same logic in a different PR (though now it seems to be gone) — worth moving to utils module?